dev/com.ibm.ws.security.saml.sso_fat.common/test-applications/samlclient.war/resources/META-INF/permissions.xml

```diff
@@ -45,5 +45,11 @@
       <actions>read</actions>
    </permission>
    
+   <permission>
```

Comment:
Why is this needed?  Are we missing a doPriv with how we are doing the serialization?

Reply:
I tried adding a doPriv where it failed in the application, but it did not resolve the issue so instead I added here like others have done. I am not quite sure why the doPriv didn't resolve it.

Reply:
Opened an issue to investigate this in the future: https://github.com/OpenLiberty/open-liberty/issues/20560